src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -298,6 +303,8 @@ private void initMockBeans(Network network) {
         when(networkStoreService.getNetwork(NETWORK_UUID)).thenReturn(network);
 
         doNothing().when(networkStoreService).deleteNetwork(NETWORK_UUID);
+
+        when(equipmentInfosService.getOrphanEquipmentInfosNetworkUuids(List.of(NETWORK_UUID))).thenReturn(ORPHAN_NETWORK_UUIDS);
```

Comment:
I think we are not testing anything here, mock "getEquipmentInfosDistinctNetworkUuids" instead ? To at least test we are actually making the exclusion ?